### PR TITLE
Don't repeat to report compile errors for logpoints and conditional bp

### PR DESCRIPTION
**Pull request by andxu**

### CODE REVIEW

File: com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java [hunk 1]
```diff
@@ -139,6 +142,16 @@ public void initialize(IDebugAdapterContext context, Map<String, Object> props)
             }
 
             if (compiledExpression.hasErrors()) {
+                if (!firstTime && breakpoint != null) {
+                    if (StringUtils.isNotBlank(breakpoint.getLogMessage())) {
+                        // for logpoint with compilation errors, don't send errors if it is already reported
+                        completableFuture.complete(debugTarget.getVM().mirrorOf(""));
```
Comment (testforstephen) [inline com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java]: IValue emptyStringValue = debugTarget.getVM().mirrorOf("");
completableFuture.complete(emptyStringValue);

File: com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java [hunk 2]
```diff
@@ -139,6 +142,16 @@ public void initialize(IDebugAdapterContext context, Map<String, Object> props)
...
+                    } else {
+                        // for conditional bp, report true to let breakpoint hit
+                        completableFuture.complete(debugTarget.getVM().mirrorOf(true));
```
Comment (testforstephen) [inline com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java]: IValue booleanValue = debugTarget.getVM().mirrorOf(true);
completableFuture.complete(booleanValue);

File: com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java (hunk 1 above)
Comment (andxu) [reply com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java]: changed

File: com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java (hunk 2 above)
Comment (andxu) [reply com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java]: changed

**Pull request merged**